packages/rule-components/src/ReportDetails/index.scss

```diff
@@ -1,13 +1,28 @@
 .ins-c-rule__report-detail {
-  padding: var(--pf-global--gutter--md) var(--pf-global--gutter);
+  padding: var(--pf-global--gutter--md) 40px;
 
   .ins-c-rule__report-detail-link {
     margin-bottom: var(--pf-global--spacer--xs);
     margin-left: var(--pf-global--spacer--form-element);
   }
 
-  .ins-c-rule__risk-detail {
+  .ins-c-rule__risk-detail-title {
     margin-bottom: var(--pf-global--spacer--sm);
+    font-weight: 200;
```

Comment:
I can't see 200 font-weights in https://patternfly-react.surge.sh/documentation/overview/global-css-variables the closest is 300 `--pf-global--FontWeight--light`

Reply:
fixed